packages/asc-ui/src/components/CustomHTMLBlock/CustomHTMLBlockStyle.ts

```diff
@@ -23,7 +23,7 @@ const ElementStyleMapping = {
   li: ListItemStyleCSS,
 }
 
-export default styled.div`
+const CustomHTMLBlock = styled.div`
```

Comment:
Then rename this to CustomHTMLBlockStyle ;)

Reply:
done!